app/stac_api/migrations/0051_update_conformance_endpoint.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.0.8 on 2024-10-09 11:45
+
+from django.db import migrations
+
+
+def add_landing_page_version(apps, schema_editor):
+    LandingPage = apps.get_model("stac_api", "LandingPage")
+    lp = LandingPage.objects.get(version='v1')
+    lp.conformsTo.insert(2, 'https://api.stacspec.org/v1.0.0/item-search')
```

Comment:
Would we also need to add conformance to https://api.stacspec.org/v1.0.0/collections/?